src/AddIns/Uno.UI.MediaPlayer.WebAssembly/HtmlMediaPlayer.NativeMethods.cs

```diff
@@ -55,6 +57,16 @@ internal static partial class NativeMethods
 			=> throw new NotSupportedException();
 #endif
 
+#if USE_JSIMPORT
+		[JSImport("globalThis.Uno.UI.Media.HtmlMediaPlayer.setPlaybackRate")]
+#endif
+		internal static partial void SetPlaybackRate(nint htmlId, double playbackRate);
+
+#if !USE_JSIMPORT
+		internal static partial void SetPlaybackRate(nint htmlId, double playbackRate)
+			=> throw new NotSupportedException();
```

Comment:
@jeromelaban : Are we supporting < NET7 ?

Reply:
We're not.